package.json

```diff
@@ -33,7 +33,7 @@
     "index.js"
   ],
   "engines": {
-    "node": ">= 0.10"
+    "node": ">= 18"
```

Comment:
Note that the space here can be omitted.

```suggestion
    "node": ">=18"
```

Reply:
sir, please.  that's my emotional support whitespace